bin/nuocd

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+#
+# (C) Copyright NuoDB, Inc. 2020-2021  All Rights Reserved
+
+# Unfortunately, ksh93 defines builtin aliases even when run non-interactively.
+unalias -a
+
+die () { echo "$*"; exit 1; }
+
+_python=${PYTHON}
+if test -z "$_python"; then
+  _python=$(command -v python2.7) || _python=$(command -v python2) || _python=$(command -v python)  \
+  || die "Python must be installed"
+fi
+
+echo "Found $_python" >&2
```

Comment:
Is this for debugging, or do we want to emit this every time we run nuocd? Perhaps we should have a more descriptive message like `Using Python executable $_python`.